knowledge-base/adding-reporting-net-core-mvc.md

```diff
@@ -0,0 +1,175 @@
+---
+title: Adding Telerik Reporting in ASP .Net Core 3.1 MVC Project
+description: Learn how to add Telerik Reporting in ASP .Net Core 3.1 MVC project.
+type: how-to
+page_title: How to Add Telerik Reporting in ASP .Net Core 3.1 MVC Project
+slug: adding-reporting-net-core-mvc
+position: 
+tags: 
+ticketid: 1454516
+res_type: kb
+---
+
+## Environment
+<table>
+	<tbody>
+		<tr>
+			<td>Product Version</td>
+			<td>14.0.20.115+</td>
+		</tr>
+		<tr>
+			<td>Product</td>
+			<td>Progress® Telerik® Reporting</td>
+		</tr>
+		<tr>
+			<td>.Net Framework</td>
+			<td>.NET Core 3.0</td>
+		</tr>
+	</tbody>
+</table>
+
+
+## Description
+This KB article lists all necessary steps for adding a Telerik Report in .NET Core MVC 3.1 project. 
```

Comment:
all necessary steps for integrating Telerik Reporting in .NET Core MVC 3.1 project.